fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java

```diff
@@ -1045,8 +1047,9 @@ public void revertCashBasedJournalEntryForSharesCharges(final Office office, fin
             createDebitJournalEntryForShares(office, currencyCode, account, shareAccountId, transactionId, transactionDate, amount);
         }
         if (totalAmount.compareTo(totalCreditedAmount) != 0) {
-            throw new PlatformDataIntegrityException("Recent Portfolio changes w.r.t Charges for shares have Broken the accounting code",
-                    "Recent Portfolio changes w.r.t Charges for shares have Broken the accounting code");
+            return; // disabling this because we broke the above rule
```

Comment:
any reason why the exception cannot be caught where this method is being called? 
if we have to return here, we need to also log a warning or we will lose track of what happens here

Reply:
In core functionality fineract doesn't support accrual accounting but we enable it so that is not required now.